docs/DeveloperGuide.adoc

```diff
@@ -448,11 +448,13 @@ image::AddmhViewmhSequenceDiagram.png[width="800"]
 
 * Command execution ends.
 
-
 ****
 NOTE: If multiple patients with the entered `NRIC` exist, then the `AddmhCommand::execute` will throw a `CommandException`
 with an appropriate error message before the use case ends.
 ****
+
+==== Design Considerations
```

Comment:
Is there any intention to add anything more here? Seems odd to have an empty region here.

Reply:
It has been filled, will update it soon!